pykickstart/commands/mount.py

```diff
@@ -0,0 +1,165 @@
+        self.mkfs_opts = kwargs.get("mkfs_opts", "")
+        self.mount_opts = kwargs.get("mount_opts", "")
+
+    def __eq__(self, other):
+        if not other:
+            return False
+
+        return self.mount_point == other.mount_point
+
+    def __ne__(self, other):
+        return not self == other
+
+    def _getArgsAsStr(self):
+        retval = "%s" % self.device
+
+        retval += " %s" % self.mount_point or "none"
+
+        if self.reformat:
+            retval += " --reformat"
+            if self.format:
+                retval += "=%s" % self.format
+
+        if self.mkfs_opts:
+            retval += " --mkfsoptions=\"%s\"" % self.mkfs_opts
+
+        if self.mount_opts:
+            retval += " --mountoptions=\"%s\"" % self.mount_opts
+
+        return retval
+
+    def __str__(self):
+        retval = BaseData.__str__(self)
+        retval += "mount %s\n" % self._getArgsAsStr()
+        return retval
+
+
+class F27_Mount(KickstartCommand):
+    """The 'mount' kickstart command"""
+
+    def __init__(self, *args, **kwargs):
+        KickstartCommand.__init__(self, *args, **kwargs)
+        self.op = self._getParser()
+        self.mount_points = kwargs.get("mount_points") or list()
+
+    def __str__(self):
+        retval = KickstartCommand.__str__(self)
+
+        for mpoint in self.mount_points:
+            retval += str(mpoint)
+
+        if retval != "":
+            retval = "# Mount points configuration\n" + retval + "\n"
+
+        return retval
+
+    def _getParser(self):
+        op = KSOptionParser()
+
+        op.add_option("--reformat", dest="reformat", default=False)
+        op.add_option("--mkfsoptions", dest="mkfs_opts", action="store")
+        op.add_option("--mountoptions", dest="mount_opts", action="store")
+
+        return op
+
+    def parse(self, args):
+        # the 'mount' command can't be used together with any other
+        # partitioning-related command
+        conflicting_command = None
+
+        # seen indicates that the corresponding
+        # command has been seen in kickstart
+        if self.handler.autopart.seen:
+            conflicting_command = "autopart"
+        if self.handler.partition.seen:
+            conflicting_command = "part/partition"
+        elif self.handler.raid.seen:
+            conflicting_command = "raid"
+        elif self.handler.volgroup.seen:
+            conflicting_command = "volgroup"
+        elif self.handler.logvol.seen:
+            conflicting_command = "logvol"
+        elif hasattr(self.handler, "reqpart") and self.handler.reqpart.seen:
+            conflicting_command = "reqpart"
+
+        if conflicting_command:
+            # allow for translation of the error message
+            errorMsg = _("The '%s' and 'mount' commands can't be used at the same time") % \
+                         conflicting_command
+            raise KickstartParseError(formatErrorMsg(self.lineno, msg=errorMsg))
```

Comment:
This handles only the case when the conflicting command is used before `mount`, no? What if it is used after `mount`?

Reply:
That's a good point. @clumens, do I need to add new version of all the ``autopart``, ``part``, ``raid``, ``volgroup``, ``logvol`` and ``reqpart`` comands just to make sure their ``parse()`` methods check that ``mount`` wasn't used together with them?

Reply:
Nah, I don't think that is necessary.  Adding the test to the older versions of commands doesn't change their behavior.

Reply:
Done.